docs/datagrid/style-by-css.md

```diff
@@ -0,0 +1,121 @@
+Style by CSS
+===
+
+By default the DataGrid is rendered as a table and all elements are styled by CSS.
+It allows you to style the DataGrid as you need. You can also add your own classes to the elements.
+
+General structure:
+```html
+<table>
+    <thead>
+        <tr>
+            <th>Column header</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Cell value</td>
+        </tr>
+    </tbody>
+</table>
+```
+
```

Comment:
We should also write here about setting the row height and column width. And about selectors for individual rows and columns.